boards/arm/lpcxpresso55s69/Kconfig.defconfig

```diff
@@ -65,3 +65,10 @@ config FLASH_LOAD_SIZE
 endif # TRUSTED_EXECUTION_NONSECURE || BOARD_LPCXPRESSO55S69_CPU1
 
 endif # BOARD_LPCXPRESSO55S69_CPU0 || BOARD_LPCXPRESSO55S69_CPU1
+
+if DMA_MCUX_LPC
+
+config HEAP_MEM_POOL_SIZE
+	default 4096
```

Comment:
Maybe add a comment about why we need a larger heap for dma.

Reply:
This config value was zero, there is a comment about this in the DTS file
https://github.com/zephyrproject-rtos/zephyr/pull/28973/files#diff-eb08125d6b85674cff6709bf72f3afb2R87

Reply:
I think it would be good to have a comment here as well.

Reply:
Sure, I will add this